change_log/next/FE-1406-carousel.yml

```diff
@@ -0,0 +1 @@
+Component Update: "Add new DLS themes to the Carousel Component"
```

Comment:
I think that should be `Styling Changes: "Adds DLS styling..."`

Reply:
The whole component changed. Maybe should I write better description?